Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/InteractiveElement.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.EventSystems;
+using UnityEngine.Serialization;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Interaction
+{
+    public class InteractiveElement : BaseInteractiveElement
+    {
+        // Place holder class
+        // A Button class will be created as an Interactive Element, the button class will 
```

Comment:
Will this be done as part of this PR, or a separate PR? I will assume separate.

Reply:
Yes this will be in a separate PR